docs/source/aryn_cloud/aryn_partitioning_service.md

```diff
@@ -118,3 +118,45 @@ You can use multiple options at the same time like in the example below:
 export ARYN_API_KEY="PUT API KEY HERE"
 curl "https://api.aryn.cloud/v1/document/partition" -o "document.json" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" -F "options={\"extract_table_structure\": true, \"threshold\": 0.2}";
 ```
+
+## Output Format
+
+The output of the Aryn Partitioning Service is json.
+
+```text
+{ "status": in-progress updates,
+  "error": any errors encountered,
+  "elements": a list of elements }
+```
```

Comment:
Do we need to explain what an element is?

Reply:
Done.